sources/Application/Views/ProjectView.cpp

```diff
@@ -114,6 +115,17 @@ ProjectView::ProjectView(GUIWindow &w,ViewData *data):FieldView(w,data) {
 	position._y+=1 ;
 	UIIntVarField *f2=new UIIntVarField(position,*v,"transpose: %3.2d",-48,48,0x1,0xC) ;
 	T_SimpleList<UIField>::Insert(f2) ;
+	
+	// from: https://github.com/xiphonics/picoTracker/blob/master/sources/Application/Views/ProjectView.cpp
```

Comment:
I'd prefer if we keep this info in the corresponding .h and .cpp, it's not necessary to have it here.